src/app/components/onc-organization/onc-organization-view.jsx

```diff
@@ -49,22 +55,8 @@ function ChplOncOrganizationView(props) {
         className={classes.header}
       />
       <CardContent className={classes.content}>
-        <Typography variant="body1" gutterBottom>
-          <strong>Organization code</strong>
-          { organization.acbCode ?? organization.atlCode }
```

Comment:
The `?? organization.atlCode` part was added by me a bit later, and supports using this component for ATLs as well as ACBs. It needs to stay in the system